txsni/dehydrated.py

```diff
@@ -0,0 +1,45 @@
+"""
+{ hostname : certificate } maps compatible with the way the
+https://dehydrated.io/ acme client stores its certificates.
+"""
+
+import pem.twisted
+
+
+class DehydratedMap(object):
+    """
+    Dehydrated's certs, in per-hostname subdirectories
```

Comment:
This seems like it's not super specific to the "dehydrated" software.

Ideally `txacme` would do the cert provisioning / challenge-response part.  Do you have a plan for how that builds on top of this?